src/doc/trpl/ownership.md

```diff
@@ -174,7 +173,7 @@ fn foo(v: Vec<i32>) -> Vec<i32> {
 }
 ```
 
-This would get very tedius. It gets worse the more things we want to take ownership of:
+This would get very tedius. It gets the worse the more things we want to take ownership of:
```

Comment:
This is incorrect, the original sentence is accurate, at least to me. Maybe this is some kind of regionalism?